src/routes/admin/mentee/mentee.route.ts

```diff
@@ -9,8 +9,10 @@ import {
 
 const menteeRouter = express.Router()
 
-menteeRouter.get('/emails/', requireAuth, getAllMenteeEmails)
-menteeRouter.get('/applications', requireAuth, getMentees)
+menteeRouter.get('/', requireAuth, getMentees)
+menteeRouter.get('/emails', requireAuth, getAllMenteeEmails)
+// TODO: Remove this route
+// menteeRouter.get('/applications', requireAuth, getMentees)
```

Comment:
can you remove this and send a PR to the frontend?
```suggestion
```

Reply:
updated